fendermint/vm/topdown/src/sync/mod.rs

```diff
@@ -176,12 +179,20 @@ fn start_syncing<T, C, P>(
     interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
 
     tokio::spawn(async move {
+        let max_proposal_range = config.max_proposal_range();
+
         let lotus_syncer =
             LotusParentSyncer::new(config, parent_proxy, view_provider, vote_tally, query)
                 .expect("");
 
         let mut tendermint_syncer = TendermintAwareSyncer::new(lotus_syncer, tendermint_client);
 
+        if let Err(e) = tendermint_syncer.init(max_proposal_range).await {
```

Comment:
Can you add commments as to why we need `max_proposal_range` blocks to be pre-filled in the cache?